inbc-program/pyproject.toml

```diff
@@ -42,3 +42,8 @@ max_tuple_unpack_length = 4
 "flake8-annotations" = ["-*", "+ANN001", "+ANN2??", "+ANN301"]
 "wemake-python-styleguide" = ["-*", "+WPS2??", "-WPS220", "-WPS223", "-WPS222", "-WPS236", "-WPS237", "-WPS238"] # WPS2?? is complexity; 220 is deep nesting; 223 is elifs; 222 is logic; 236 is # of variables unpacked in a tuple; 237 is complex format string; 238 is # of raises in a function
 "flake8-bandit" = ["+*", "-S404", "-S603"]
+
+[tool.pytest.ini_options]
```

Comment:
do we want this to be enabled in all pyproject.toml files or just inbc-program?

Reply:
we probably want it in all of them.  I found I couldn't debug the tests very well without it.  I can add it to the others, so someone else doesn't run into the same issue with another component.